### DISCUSSION TITLE
Docker support?

### DISCUSSION THREAD

**Discussion by uhlhosting** [pschlan/cron-job.org#196] [Q&A]
Hi, any plans to add docker support or precompiled packages?

Right now as I see it, the installation steps are ... maybe good for the person who made it.

**Comment by uhlhosting** [comment]
@pschlan can you try to build images ONLY for the services part of the cron-job, and leave the rest needed out, for the users to connect to them? 

Such as redis, nginx, and anything that is not part of the app itself. In that way, I could pull the docker image from docker hub, and add own redis version, nginx or anything else needed. 

Id like to use a platform like Caprover to make my template. So there apps should be having their own image in docker hub for easiest deployment, there is also manual deployment, yet thats not very practical when having to deploy it several times and test it properly.

**Comment by Nurullaev** [comment]


**Reply by uhlhosting** [reply]
@Nurullaev english?
